src/targets/__tests__/upm.test.ts

```diff
@@ -0,0 +1,66 @@
+import { NoneArtifactProvider } from '../../artifact_providers/none';
+import { UpmTarget } from '../upm';
+
+describe('UPM Target', () => {
+  const cleanEnv = { ...process.env };
+  let upmTarget: UpmTarget;
+
+  beforeEach(() => {
+    process.env = {
+      ...cleanEnv,
+      GITHUB_TOKEN: 'ghp_TAN21WJQkLtYVdHCh5eQJ8hTWoYvNh47gGWH',
+      DRY_RUN: '',
```

Comment:
Will verify that `'false'` actually disables it and use that :)

Reply:
It does - I checked. Wouldn't have made the suggestion otherwise! 🙂 

https://github.com/getsentry/craft/blob/c8da1e575a1579e1cee4c33a1d6b082fe4979eb9/src/utils/helpers.ts#L1-L12